README.md

```diff
@@ -177,28 +177,12 @@ target platform whenever dependencies are updated.
 
   0. As described above, you must first build the target platform with Maven:
   
-     `$ mvn -Pide-target-platform package`
+     `$ (cd eclipse; mvn package)`
      
-  1. Open the `Preferences` dialog, go to `Plug-in Development` > `Target Platform`.
+  1. Open the `ide-target-platform/gcp-ide-target-platform.target` file.
```

Comment:
Sorry, I should have mentioned this:  I renamed the `eclipse/ide-target-platform` project to just `ide-target-platform`.  The Maven artifactID was previously `ide-target-platform.repo` and the `.project` had `maven-dependencies`.  You may need to delete the `maven-dependencies` and/or `ide-target-platform.repo` projects and re-import them as existing projects.